serialization/aut/src/main/java/net/automatalib/serialization/aut/InternalAUTParser.java

```diff
@@ -98,9 +99,9 @@ private void parseHeader(BufferedReader reader) throws IOException {
         verifyLBracketAndShift();
         initialState = parseNumberAndShift();
         verifyCommaAndShift();
-        parseNumberAndShift(); // ignore number of states
-        verifyCommaAndShift();
         parseNumberAndShift(); // ignore number of transitions
+        verifyCommaAndShift();
+        numStates = parseNumberAndShift(); // store number of states
```

Comment:
Since you now parse the number of states (in the correct order ;), could you also add a check that this number is greater or equal to 1, like required by the [specification](https://cadp.inria.fr/man/aut.html#sect2)? Also adding a test case for this behavior (which is expected to fail) would be appreciated.

Reply:
Throwing a simple `IllegalArgumentException` should be sufficient.